hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServletInodeBasedXfer.java

```diff
@@ -258,6 +260,17 @@ public void writeDbDataToStream(HttpServletRequest request, OutputStream destina
         writeDBToArchive(sstFilesToExclude, checkpointDir,
             maxTotalSstSize, archiveOutputStream, tmpdir, hardLinkFileMap, false);
         if (includeSnapshotData) {
+          // get the list of snapshots from the checkpoint
+          OmMetadataManagerImpl checkpointMetadataManager = null;
+          try {
```

Comment:
don't we need to acquire SNAPSHOT_LOCK here?

Reply:
we are reading it from the checkpoint and already have the bootstrap lock held so it's not needed.